### PR TITLE
Delete deprecated commands and flags

### LINKED ISSUE #876: Delete deprecated commands and flags
### Deprecated flags per command:
skupper init
- cluster-local
- edge
- xp-router-max-frame-size
- xp-router-max-session-frames

skupper gateway init
- exportonly
- name

skupper gateway delete
- name

skupper gateway download
- name

skupper gateway export-config
- name

skupper gateway expose
- name

skupper gateway unexpose
- name

skupper gateway bind
- name
- protocol

skupper gateway unbind
- name
- protocol

skupper gateway forward
- name
- protocol

skupper gateway unforward
- name
- protocol

### Deprecated commands:
- skupper bind
- skupper unbind
- skupper list-connectors
- skupper connect
- skupper disconnect
- skupper check-connection
- skupper connection-token
- skupper list-exposed
- skupper gateway download

### PR DESCRIPTION
**Pull request by nluaces**
Fixes #876

### CODE REVIEW

**Review by grs — APPROVED** [review]

**Review by fgiorgetti — APPROVED** [review]
Great job @nluaces

**Review by ajssmith — APPROVED** [review]
lgtm, thanks!

File: cmd/skupper/skupper_cluster_test.go [hunk 1]
```diff
@@ -494,13 +494,13 @@ func TestConnectWithInteriorCluster(t *testing.T) {
 			doc:             "connect-test1",
 			args:            []string{"--help"},
 			expectedCapture: "",
-			expectedOutput:  "Connect this skupper installation to that which issued the specified connectionToken",
+			expectedOutput:  "Links this skupper installation to that which issued the specified token",
```
Comment (pwright) [inline cmd/skupper/skupper_cluster_test.go:497]:
```suggestion
			expectedOutput:  "Links this skupper site to the site that issued the token",
```

**Review by pwright — APPROVED** [review]
just one suggested text change

**Pull request merged**